cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -68,12 +71,28 @@ sub execute {
         $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
         return $self->error("Database error: " . $dbh->errstr)
             if $dbh->err;
+        if ( $alias ne $alias_with_dashes ) {
+            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
```

Comment:
As I understand it, you'd still have both abc-123 and abc_123 aliases in the table after renaming abc_123 to abc, unless something else caused them to be deleted. The update_email_alias call would only affect rows where the alias (first column) is 'abc'.